### PR TITLE
Wb/account info

### PR DESCRIPTION
**Pull request by joebrashear**
This has everything that is necessary for the project, minus the API functionality

### CODE REVIEW

File: app/lib/navbar.dart [hunk 1]
```diff
@@ -232,4 +247,11 @@ class _NavbarState extends State<Navbar> {
...
-}
\ No newline at end of file
+
+  static void takePhoto(ImageSource source) async {
+    ImagePicker picker;
+    final pickedFile = await picker.getImage(
+      source: source,
+    );
+  }
+}
```
Comment (pstoebenau) [inline app/lib/navbar.dart]:
```suggestion

```

Comment (pstoebenau) [reply app/lib/navbar.dart]: Why is this here?

File: app/lib/snippet-view.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// class _SnippetViewState extends State<SnippetView> {
+
+//   _SnippetViewState(this.snippet, {Object snippet});
+//   @override
+//   Widget build(BuildContext context) {
+//     return Container(
+      
+//     );
+//   }
+// }
```
Comment (pstoebenau) [inline app/lib/snippet-view.dart:21]: Why is this commented out?

**Review by pstoebenau — CHANGES_REQUESTED** [review]

File: app/lib/navbar.dart (hunk 1 above)
Comment (joebrashear) [reply app/lib/navbar.dart]: No need anymore, will delete

File: app/lib/snippet-view.dart (hunk 2 above)
Comment (joebrashear) [reply app/lib/snippet-view.dart:21]: This is not finished yet, and if I don't comment it out it will give everyone build errors

Comment (pstoebenau) [reply app/lib/snippet-view.dart:21]: ok

**Review by pstoebenau — APPROVED** [review]

**Pull request merged**